scheduler/src/db.rs!

```diff
@@ -0,0 +1,98 @@
+use crate::{Error, Result};
```

Comment:
It is file correct?  (notice ! at the end)

Reply:
yes, this file should be removed.